packages/agent-app-sdk/src/widgets/settings/settings-widget.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  createWidget,
+  withAmplitude,
+  createConnection,
+  IConnection
+} from '@livechat/widget-core-sdk';
+import { ISettingsWidgetApi, ISettingsWidgetEvents } from './interfaces';
+
+export function SettingsWidget(
+  connection: IConnection<ISettingsWidgetEvents>
+) {
+  const base = createWidget<ISettingsWidgetApi, ISettingsWidgetEvents>(
+    connection,
+    {
+      redirect(target: string): Promise<void> {
+        return connection.sendMessage(
+          'settings_widget_redirect',
```

Comment:
I thought about naming the vent `redirect`, based on current events:
https://github.com/livechat/my.livechatinc.com/blob/develop/source/tsx/services/widget/message/inbox/inbox-message-titles.ts

But as we want to limit it to settings widgets only, I decided to use a less generic name. WDYT?

Reply:
changed in [d56c82b](https://github.com/livechat/products-sdk/pull/22/commits/d56c82bd8e092679fd41d87dd5edd53b26482c8d)